js/test/transactions.spec.js

```diff
@@ -0,0 +1,169 @@
+/** ******************************************************************************
+ *  (c) 2020 Zondax GmbH
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ ******************************************************************************* */
+
+import FlowApp from "../src";
+import TransportNodeHid from "@ledgerhq/hw-transport-node-hid";
+import chai, { expect } from "chai"
+import jsSHA from "jssha";
+import {ec as EC} from "elliptic";
+import fs from "fs";
+
+jest.setTimeout(1000000);
```

Comment:
When running a test dealing with the real Nano S device, is it possible to get screen shots from the real device? I assume it is not possible. If it is not possible, why use `jest` which is for comparing and managing screen shots?

Reply:
AFAIK (new to js) jest is general purpose testing framework. I see little reason to use something other then zemu_tests use.

Reply:
AFAIK (I am new to js), jest is general purpose testing framework. While running these test you see what is going on on the device.

Reply:
Yep, I'm also new to js and think I got it confused with another module :-)